content/post/monitor-pgpool-with-grafana-dashboard-in-amazon-eks/index.md

```diff
@@ -0,0 +1,497 @@
+pgpool.kubedb.com/pgpool   kubedb.com/v1alpha2   4.5.0     Ready    2m24s
+```
+
+> We have successfully deployed Pgpool in Amazon EKS.
+
+### Import Grafana Dashboard
+Here, we will port-forward the `prometheus-grafana` service to access Grafana Dashboard from UI.
+
+```bash
+$ kubectl get service -n monitoring
+NAME                                      TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)                      AGE
+alertmanager-operated                     ClusterIP   None            <none>        9093/TCP,9094/TCP,9094/UDP   23m
+prometheus-grafana                        ClusterIP   10.96.233.2     <none>        80/TCP                       25m
+prometheus-kube-prometheus-alertmanager   ClusterIP   10.96.14.231    <none>        9093/TCP,8080/TCP            25m
+prometheus-kube-prometheus-operator       ClusterIP   10.96.248.189   <none>        443/TCP                      25m
+prometheus-kube-prometheus-prometheus     ClusterIP   10.96.121.184   <none>        9090/TCP,8080/TCP            25m
+prometheus-kube-state-metrics             ClusterIP   10.96.119.64    <none>        8080/TCP                     25m
+prometheus-operated                       ClusterIP   None            <none>        9090/TCP                     23m
+prometheus-prometheus-node-exporter       ClusterIP   10.96.14.107    <none>        9100/TCP                     25m
+```
+To access Grafana UI Let's port-forward `prometheus-grafana` service to 3063 
+
+```bash
+$ kubectl port-forward -n monitoring service/prometheus-grafana 3063:80
+Forwarding from 127.0.0.1:3063 -> 3000
+Forwarding from [::1]:3063 -> 3000
+
+```
+Now, go to `http://localhost:3063/` in your web browser. You will see the Grafana login screen. Use the default credentials: `admin` for the `Username` and `prom-operator` for the `Password`.
+
+![Grafana Login](grafana-login.png)
+
+Once logged in, import the Pgpool dashboards by clicking on the `Import` button in the left sidebar of the Grafana UI.  
+
+![Import Dashboard](import-dashboard.png)
+
+You can either upload a JSON file or copy and paste the JSON content from the available dashboard files. Visit the [Pgpool Dashboards](https://github.com/ops-center/grafana-dashboards/tree/master/pgpool) to find the required JSON files. Upload the JSON file or paste the content into the panel JSON field and click `Load` to import the dashboard.
+
+![Upload JSON](upload-json.png)
+
+After completing these steps, you should see the Pgpool Grafana Dashboards in your Grafana UI. Here are some screenshots of the Pgpool deployment:
+
+![Sample UI 1](sample-ui-1.png)
+
+![Sample UI 2](sample-ui-2.png)
+
+![Sample UI 3](sample-ui-3.png)
+
+> Congratulations! You have successfully set up Pgpool monitoring with Grafana Dashboards using KubeDB.
+
+
+We have made an in depth tutorial on Seamlessly Provision and Manage Pgpool on Kubernetes Using KubeDB. You can have a look into the video below:
+
```

Comment:
https://www.youtube.com/watch?v=-0_2I5dR6CM
This video seems to have more accurate usage for this blog?